osquery/tables/system/darwin/nfs_shares.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#include <vector>
+#include <string>
+
+#include <boost/algorithm/string/join.hpp>
+#include <boost/algorithm/string/predicate.hpp>
+
+#include <glog/logging.h>
+
+#include <osquery/core.h>
+#include <osquery/tables.h>
+#include <osquery/filesystem.h>
+
+namespace osquery {
+	namespace tables {
+
+		QueryData parseNfsSharesContent(const std::string& content) {
+			QueryData results;
+
+			for (const auto& i : split(content, "\n")) {
+				auto line = split(i);
+				if (line.size() == 0 || boost::starts_with(line[0], "#")) {
+					continue;
+				}
+				std::vector<std::string> lineExports;
+				unsigned int readonly = 0;
+				int indexOfOptions = -1;
+
+				for (std::vector<std::string>::iterator iter = line.begin();
+					 iter != line.end();
+					 ++iter){
+					indexOfOptions++;
+					if((*iter)[0] == '/'){
+						lineExports.push_back(*iter);
+						
+					}else{
+						break;
+					}
+				}
+				for(std::vector<std::string>::iterator iter = line.begin()+indexOfOptions;
+					 iter != line.end();
+					 ++iter){
+					if(iter->compare("-ro") == 0|| iter->compare("-o") == 0){
+						readonly = 1;
+					}
+				}
+				for (std::vector<std::string>::iterator iter = lineExports.begin(); iter != lineExports.end(); ++iter)
+				{
+					Row r;
+					r["share"] = *iter;
+					if(readonly){
+						r["readonly"]  = "yes";
```

Comment:
I suggested a "1" or "0" since we've more-or-less converged on integer representation for booleans. It's most friendly to SQL if the column is marked as a INTEGER type too!

Since there's no boolean type:
```cpp
SQLITE_API const void *sqlite3_column_blob(sqlite3_stmt*, int iCol);
SQLITE_API int sqlite3_column_bytes(sqlite3_stmt*, int iCol);
SQLITE_API int sqlite3_column_bytes16(sqlite3_stmt*, int iCol);
SQLITE_API double sqlite3_column_double(sqlite3_stmt*, int iCol);
SQLITE_API int sqlite3_column_int(sqlite3_stmt*, int iCol);
SQLITE_API sqlite3_int64 sqlite3_column_int64(sqlite3_stmt*, int iCol);
SQLITE_API const unsigned char *sqlite3_column_text(sqlite3_stmt*, int iCol);
SQLITE_API const void *sqlite3_column_text16(sqlite3_stmt*, int iCol);
SQLITE_API int sqlite3_column_type(sqlite3_stmt*, int iCol);
SQLITE_API sqlite3_value *sqlite3_column_value(sqlite3_stmt*, int iCol);
```